### PR TITLE
PD-596 mention pool ID, node IP

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/reference/CLI/create-and-manage-volumes/_index.md [hunk 1]
```diff
@@ -177,6 +177,10 @@ These options can also be passed in through the scheduler or using the inline vo
 
 Use the `--nodes=LocalNode` flag to create a volume and place at least one replica of the volume on the node where the command is run. This is useful when you use a script to create a volume locally on a node.
 
+{{<info>}}
+**NOTE:** You can provide a node ID, node IP address, or pool UUID to the `--nodes` flag.
```
Comment (nathan-px) [inline content/reference/CLI/create-and-manage-volumes/_index.md:181]: @sangleganesh Not sure if this is where the note should go for volume create.

File: content/reference/CLI/updating-volumes.md [hunk 2]
```diff
@@ -289,131 +289,146 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
-Here is the output of `pxctl alerts show --type volume`:
-
-```text
-25	970758537931791410	Feb 26 22:02:04 UTC 2017	NOTIFY		Volume operation success	Volume (Id: 970758537931791410 Name: clitest) HA updated from 1 to 2
-26	970758537931791410	Feb 26 22:58:17 UTC 2017	NOTIFY		Volume operation success	Volume (Id: 970758537931791410 Name: clitest) HA updated
-```
+The `ha-update` command can be used to reduce the replication factor as well. Follow the instructions below to decrease a volume's replication factor and remove replicas from a node or storage pool:
+
+
+1. Begin removal of your volume's replica from your target node or storage pool by entering the following `pxctl volume ha-update` command, specifying:
```
Comment (andreipope) [inline content/reference/CLI/updating-volumes.md:381]:
```suggestion
1. Sequentially remove one of your volume's replica from your target node or storage pool by entering the following `pxctl volume ha-update` command, specifying:
```

File: content/reference/CLI/updating-volumes.md [hunk 3]
```diff
@@ -289,131 +289,146 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
+
+	* `--repl=` with the number of replicas you want to reduce to. This must be lower than your volume's current replication factor.
```
Comment (andreipope) [inline content/reference/CLI/updating-volumes.md:383]:
```suggestion
	* `--repl=` with the new number of replicas. This must be equal to your volume's current replication factor minus one.
```

File: content/reference/CLI/updating-volumes.md [hunk 4]
```diff
@@ -289,131 +289,146 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
+	--repl=1 \
+	--node b1aa39df-9cfd-4c21-b5d4-0dc1c09781d8 \
+	<volume-name>
+	```
+	```output
+	Update Volume Replication: Replication update started successfully for volume exampleVolume
+	```
+
+2. Monitor the replication operation by entering the following `pxctl alerts show` command:
+
```
Comment (andreipope) [inline content/reference/CLI/updating-volumes.md:398]:
```suggestion
    ```text
    pxctl alerts show --type volume
    ```
```

File: content/reference/CLI/updating-volumes.md [hunk 5]
```diff
@@ -289,131 +289,146 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
+	```text
```
Comment (andreipope) [inline content/reference/CLI/updating-volumes.md:399]:
```suggestion
	```output
```

File: content/reference/CLI/updating-volumes.md [hunk 6]
```diff
@@ -289,131 +289,146 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
+	ID					DATA IP		CPU		MEM TOTAL	MEM FREE	CONTAINERS	VERSION		STATUS
+	fa18451d-9091-45b4-a241-d816357f634b	10.99.117.133	0.5		8.4 GB	7.9 GB		N/A		1.1.6-a879596	Online
+	b1aa39df-9cfd-4c21-b5d4-0dc1c09781d8	10.99.117.137	0.250313	8.4 GB	7.9 GB		N/A		1.1.6-a879596	Online
+	bb605ca6-c014-4e6c-8a23-55c967d1a963	10.99.117.135	0.625782	8.4 GB	7.9 GB		N/A		1.1.6-a879596	Online
+	```
+
+
+2. Begin replicating your volume to your target node or storage pool by entering the following `pxctl volume ha-update` command, specifying:
+
+	* `--repl=` with the number of replicas you want to create. This must be higher than your volume's current replication factor.
```
Comment (andreipope) [inline content/reference/CLI/updating-volumes.md:320]:
```suggestion
	* `--repl=` with the new number of replicas you want to create. This must be equal to your volume's current replication factor plus one.
```

File: content/reference/CLI/updating-volumes.md [hunk 7]
```diff
@@ -289,131 +289,148 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
+	fa18451d-9091-45b4-a241-d816357f634b	10.99.117.133	0.5		8.4 GB	7.9 GB		N/A		1.1.6-a879596	Online
+	b1aa39df-9cfd-4c21-b5d4-0dc1c09781d8	10.99.117.137	0.250313	8.4 GB	7.9 GB		N/A		1.1.6-a879596	Online
+	bb605ca6-c014-4e6c-8a23-55c967d1a963	10.99.117.135	0.625782	8.4 GB	7.9 GB		N/A		1.1.6-a879596	Online
+	```
+
+
+2. Begin replicating your volume to your target node or storage pool by entering the following `pxctl volume ha-update` command, specifying:
+
+	* `--repl=` with the new number of replicas you want to create. This must be equal to your volume's current replication factor plus one.
+	* `--node` with the node ID, node IP address, or pool UUID you want to create the replica(s) on.
```
Comment (nathan-px) [inline content/reference/CLI/updating-volumes.md:321]: @sangleganesh I made a lot of formatting changes to this document, but the only material change was the addition of the node IP address and pool UUID here. Can you just verify and approve based on whether this is correct?

File: content/reference/CLI/updating-volumes.md [hunk 8]
```diff
@@ -289,131 +289,148 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
+	* The volume you want to increase the replication factor for.
+
+	```text
+	pxctl volume ha-update \
+	--repl=2 \
+	--node <node-ID>
```
Comment (sangleganesh) [inline content/reference/CLI/updating-volumes.md:327]: may be node-ID|pool-uuid|node-IP

File: content/reference/CLI/updating-volumes.md [hunk 9]
```diff
@@ -289,131 +289,148 @@ For more information about dynamically resizing a volume (PVC) using Kubernetes
...
+1. Begin removal of your volume's replica from your target node or storage pool by entering the following `pxctl volume ha-update` command, specifying:
+
+	* `--repl=` with the new number of replicas. This must be equal to your volume's current replication factor minus one.
+	* `--node` with the node ID, node IP address, or pool UUID you want to remove a replica from.
+	* The volume you want to decrease the replication factor for.
+
+	```text
+	pxctl volume ha-update  \
+	--repl=1 \
+	--node b1aa39df-9cfd-4c21-b5d4-0dc1c09781d8 \
```
Comment (sangleganesh) [inline content/reference/CLI/updating-volumes.md:390]: node-ID|node-IP|pool-UUID ?

**Pull request merged**